src/renderer/components/helpers/OpenPGP4FPRUrl.ts

```diff
@@ -0,0 +1,126 @@
+import { DeltaBackend } from '../../delta-remote'
```

Comment:
Naming of this file is a bit strange: `OpenPGP4FPRUrl.ts` as the name only refers to a part of this file